README.md

```diff
@@ -536,6 +536,48 @@ local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') + {
 { ['grafana-' + name]: kp.grafana[name] for name in std.objectFields(kp.grafana) }
 ```
 
+#### Defining the ServiceMonitor for each addional Namespace
+
+In order to Prometheus be able to discovery and scrap services inside the additional namespaces specified in previous step you need to define a ServiceMonitor resource.
+
+You can define ServiceMonitor resources in your `jsonnet` spec. See the snippet bellow:
+
+```
```

Comment:
if you use the same style as the other snippets (as in the jsonnet file in the `examples/` directory), then this even gets built on CI, so we know it works :wink:

Reply:
perfect!

Reply:
done!